library/src/app/components/account-list/account-list.component.ts

```diff
@@ -0,0 +1,149 @@
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import {
+  BehaviorSubject,
+  catchError,
+  map,
+  of,
+  Subject,
+  switchMap,
+  takeUntil,
+  timer
+} from 'rxjs';
+import { MatTableDataSource } from '@angular/material/table';
+
+// Services
+import {
+  AccountService,
+  Account,
+  ComponentConfig,
+  LEVEL,
+  CODE,
+  ConfigService,
+  EventService,
+  ErrorService,
+  RoutingService
+} from '@services';
+
+// Utility
+import { Constants } from '@constants';
+import { MatSort } from '@angular/material/sort';
+
+@Component({
+  selector: 'app-account-list',
+  templateUrl: './account-list.component.html',
+  styleUrls: ['./account-list.component.scss']
+})
+export class AccountListComponent implements OnInit, OnDestroy {
+  @ViewChild(MatSort) set matSort(sort: MatSort) {
+    this.dataSource.sort = sort;
+    this.dataSource.sortingDataAccessor = this.sortingDataAccessor;
+  }
+
+  balance$ = new BehaviorSubject<number>(0);
+
+  isLoading$ = new BehaviorSubject(true);
+  isRecoverable$ = new BehaviorSubject(true);
+  private unsubscribe$ = new Subject();
+
+  // todo(Dustin) look into getting counter_asset as part of the config
+  counter_asset = Constants.USD_ASSET;
```

Comment:
How are we going to handle banks that support multiple fiat assets?

Should we support a list here that is possibly empty? To handle cases where:

a) the bank does not want to show the fiat value
b) the bank wants to show the fiat value in a single fiat asset, e.g., CAD
c) the bank wants to show the fiat value in multiple fiat assets, e.g., CAD and USD

Reply:
This points towards adding a fiat param to the application config.

Reply:
This should be a config to start. Defaulting to USD is fine, but let's build in as a single value (that if not set does not display).

Please coordinate with @nalancer08 and @jc-cybrid as well as @faramarz-cybrid

Reply:
I've added it to the component config as a required param, default to 'USD'. We can discuss in the SDK meeting.